msteams-platform/graph-api/meeting-transcripts/get-transcripts-api.md

```diff
@@ -0,0 +1,880 @@
+
+    </details>
+
+2. **Get meeting ID and organizer ID**: Use `joinUrl` to get the meeting details to fetch transcript.
+
+    Use the following example to request the online meeting ID using the URL:
+
+    ```http
+    GET https://graph.microsoft.com/beta/me/onlineMeetings?$filter=joinUrl%20eq%20'https%3A%2F%2Fteams.microsoft.com%2Fl%2Fmeetup-join%2F19%253ameeting_MGQ4MDQyNTEtNTQ2NS00YjQxLTlkM2EtZWVkODYxODYzMmY2%2540thread.v2%2F0%3Fcontext%3D%257b%2522Tid%2522%253a%2522909c6581-5130-43e9-88f3-fcb3582cde37%2522%252c%2522Oid%2522%253a%2522dc17674c-81d9-4adb-bfb2-8f6a442e4622%2522%257d'
+    ```
+
+    The response payload contains:
+    - Meeting ID in  `value` > `id`.
+    - Organizer ID in `participants` > `organizer` > `identity` > `user` > `id`.
+    <br>
+    <details>
+    <summary><b>Example</b>: Response payload with meeting ID</summary>
+
+    ```json
+    {
+    "value": [
+        {
+            "id": "dc17674c-81d9-4adb-bfb2-8f6a442e4622_19:meeting_MGQ4MDQyNTEtNTQ2NS00YjQxLTlkM2EtZWVkODYxODYzMmY2@thread.v2",
+            "creationDateTime": "2020-09-29T22:35:33.1594516Z",
+            "startDateTime": "2020-09-29T22:35:31.389759Z",
+            "endDateTime": "2020-09-29T23:35:31.389759Z",
+            "joinWebUrl": "https://teams.microsoft.com/l/meetup-join/19%3ameeting_MGQ4MDQyNTEtNTQ2NS00YjQxLTlkM2EtZWVkODYxODYzMmY2%40thread.v2/0?context=%7b%22Tid%22%3a%22909c6581-5130-43e9-88f3-fcb3582cde37%22%2c%22Oid%22%3a%22dc17674c-81d9-4adb-bfb2-8f6a442e4622%22%7d",
+            "subject": null,
+            "autoAdmittedUsers": "EveryoneInCompany",
+            "isEntryExitAnnounced": true,
+            "allowedPresenters": "everyone",
+            "allowMeetingChat": "enabled",
+            "allowTeamworkReactions": true,
+            "videoTeleconferenceId": "(redacted)",
+            "participants": {
+                "organizer": {
+                    "upn": "(redacted)",
+                    "role": "presenter",
+                    "identity": {
+                        "user": {
+                            "id": "dc17674c-81d9-4adb-bfb2-8f6a442e4622",
+                            "displayName": null,
+                            "tenantId": "909c6581-5130-43e9-88f3-fcb3582cde38",
+                            "identityProvider": "AAD"
+                        }
+                    }
+                },
+                "attendees": [],
+                "producers": [],
+                "contributors": []
+            },
+            "lobbyBypassSettings": {
+                "scope": "organization",
+                "isDialInBypassEnabled": false
+            }
+        }
+    ]
+    }
+    ```
+
+    </details>
+
+After your app obtains the meeting ID and the organizer ID, it triggers the Graph APIs to fetch transcript content using these meeting details.
+
+## Use Graph APIs to fetch transcript
```

Comment:
@mankadnandan I think we should reference the API docs that you worked on out here instead of copying over examples here in this scenario doc too.

Having the same thing documented in two places makes it harder to maintain. What do you think?

Reply:
So currently the public docs are not merged and live. So, we might not want to share the PR under review yet. We can surely reference the link to the public docs here later once they are live.

Reply:
Let's get the API reference docs merged first then, followed by referencing them here